Dockerfile

```diff
@@ -27,9 +27,12 @@ COPY --chown=1001:1001 --from=hapi-distroless /app /app
 
 COPY --chown=1001:1001 --from=bun  /usr/local/bin/bun /usr/bin/
 ADD --chown=1001:1001 ./server-setup /bonfhir/server-setup/
+ADD --chown=1001:1001 ./plugins /bonfhir/plugins/
 
 USER bonfhir
 RUN cd /bonfhir/server-setup && bun install
+RUN cd /bonfhir/plugins/rxnorm && bun install
+RUN cd /bonfhir/plugins/cpt-hcpcs && bun install
```

Comment:
have you looked into [https://bun.sh/docs/install/workspaces](https://bun.sh/docs/install/workspaces) instead of performing separate plugin installs?

Reply:
✅